README.md

```diff
@@ -60,25 +76,41 @@ spec:
     type: OpenShiftSDN
   serviceNetwork: 172.30.0.0/16
 ```
+For OVN it is:
+```yaml
+apiVersion: networkoperator.openshift.io/v1
+kind: NetworkConfig
+metadata:
+  name: default
+spec:
+  additionalNetworks: null
+  clusterNetworks:
+  - cidr: 10.128.0.0/14
+    hostSubnetLength: 9
+  defaultNetwork:
+    type: OVNKubernetes
+  serviceNetwork: 172.30.0.0/16
+```
+The cidr, hostSubnetLength is internally converted into the cidr/hostPrefix etended cidr used by OVN, 10.128.0.0/14/23.
 
 ## Configuring IP address pools
-Users must supply at least two address pools - one for pods, and one for services. These are the ClusterNetworks and ServiceNetwork parameter. Some network plugins, such as OpenShiftSDN, support multiple ClusterNetworks. All address blocks must be non-overlapping. You should select address pools large enough to fit your anticipated workload.
+Users must supply at least two address pools - one for pods, and one for services. These are the ClusterNetworks and ServiceNetwork parameter. Some network plugins, such as OpenShiftSDN, and OVNKubernetes, support multiple ClusterNetworks. All address blocks must be non-overlapping. You should select address pools large enough to fit your anticipated workload. Each pool must be able to hold 1 or more hostPrefix allocations.
 
 For future expansion, multiple `serviceNetwork` entries are allowed by the configuration but not actually supported by any network plugins. Supplying multiple addresses is invalid.
 
-Each `clusterNetwork` entry has an additional required parameter, `hostPrefix`, that specifies the address size to assign to assign to each individual node.  For example,
+Each `clusterNetwork` entry has an additional required parameter, `hostPrefix`, that specifies the address size to assign to each individual node.  For example,
 ```yaml
 cidr: 10.128.0.0/14
 hostPrefix: 23
 ```
-means nodes would get blocks of size `/23`, or 512 addresses.
+means nodes would get blocks of size `/23`, or 512 addresses. This is converted into hostSubnetLength in the networkoperator.
 
-IP address pools are always read from the Cluster configuration and propagated "downwards" in to the Operator configuration. Any changes to the Operator configuration will be ignored.
+IP address pools are always read from the Cluster configuration and propagated "downwards" into the Operator configuration. Any changes to the Operator configuration are ignored.
 
 Currently, changing the address pools once set is not supported. In the future, some network providers may support expanding the address pools.
 
 
-Example
+OpenShiftSDN or OVNKubernetes Example:
```

Comment:
We've already said that not all plugins support multiple cidrs. So just leave this as "Example"